src/agent/src/confidential_data_hub/mod.rs

```diff
@@ -101,10 +114,31 @@ impl CDHClient {
         };
         let res = self
             .get_resource_client
-            .get_resource(ttrpc::context::with_timeout(*CDH_API_TIMEOUT), &req)
+            .get_resource(
+                ttrpc::context::with_timeout(AGENT_CONFIG.cdh_api_timeout.as_nanos() as i64),
+                &req,
+            )
             .await?;
         Ok(res.Resource)
     }
+
+    pub async fn pull_image(&self, image: &str, bundle_path: &str) -> Result<String> {
+        let req = confidential_data_hub::ImagePullRequest {
+            image_url: image.to_string(),
+            bundle_path: bundle_path.to_string(),
+            ..Default::default()
+        };
+
+        let res = self
+            .image_pull_client
+            .pull_image(
+                ttrpc::context::with_timeout(AGENT_CONFIG.cdh_api_timeout.as_nanos() as i64),
```

Comment:
Imho there should be different timeouts for pulling images and other CDH operations, considering that an image pull might take a significantly long time to complete.

Reply:
Yes. This original commit uses a common timeout configuration of CDH, and [this commit](https://github.com/kata-containers/kata-containers/pull/11197/commits/0c52adb1954dbc6289aaedc034e80637c64e7891#diff-f74876ae6254fdc85d8dd843c9c37af6ec76c1a6da631604d85be763381513c6) specialize it with another.